packages/utilities/src/hmac.ts

```diff
@@ -0,0 +1,37 @@
+import crypto from 'crypto';
+
+const CHARSET = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'.split('');
+
+/**
+ * Encodes BigInt to base62.
+ * To avoid new dependency, this function was copied from https://github.com/base62/base62.js/blob/master/lib/ascii.js
```

Comment:
```suggestion
```
You can remove this comment now, since it's basically a new function 🙂 And we won't have to worry about including the license of that package.